src/vs/workbench/browser/parts/notifications/notificationsCenter.ts

```diff
@@ -314,8 +321,15 @@ export class NotificationsCenter extends Themable implements INotificationsCente
 			}
 		}
 	}
+
+	onDidSetDoNotDisturbMode(mode: NotificationsDoNotDisturbMode): void {
```

Comment:
* `private` is missing
* maybe move method below the `onDidChangeNotification` to move event handling closer to each other.